projects/hslayers/src/common/get-capabilities/wms-get-capabilities-response.interface.ts

```diff
@@ -114,7 +114,15 @@ export type WmsDimension = {
   current?: boolean;
   values: string | string[];
 };
-export type WmsLayer = {
+
+/***
+ * HsLayers extension of default WMS Layer object
+ */
+export type HsLayersWmsLayer = {
```

Comment:
I would defined the types the other way around...

Reply:
I was thinking about it as a hslayers extension to the base response, but yes this makes more sense. Thanks for the change